services/convert/user.go

```diff
@@ -59,13 +55,31 @@ func toUser(ctx context.Context, user *user_model.User, signed, authed bool) *ap
 		Website:     user.Website,
 		Description: user.Description,
 		// counter's
-		Followers:    user.NumFollowers,
-		Following:    user.NumFollowing,
 		StarredRepos: user.NumStars,
 	}
 
 	result.Visibility = user.Visibility.String()
 
+	authed := false
+	signed := false
+	if doer != nil {
+		signed = true
+		authed = doer.ID == user.ID || doer.IsAdmin
+
+		_, followersCount, err := user_model.GetUserFollowers(ctx, user, doer, db.ListOptions{ListAll: true})
+		if err != nil {
+			return nil
```

Comment:
Not sure whether it is right here.
Same in L76.

Reply:
We should not find all followers here, we just need a number. It will waste unnecessary CPU and Memory.

Reply:
What do you mean `we just need a number`.
Just set a fake number, or set `Page = 1` and `PageSize = xx` in ListOptions.